setup.py

```diff
@@ -40,7 +40,7 @@
         "imageio-ffmpeg~=0.4.9",
         "termcolor~=2.4.0",
         "progressbar~=2.5",
-        "pymeshlab~=2022.2",
```

Comment:
I don't think this will works with the unit tests? (cc. @cgokmen )

Reply:
reverted back. could you give me an approval?